README.md

```diff
@@ -1 +1,69 @@
-# mobile-menu
\ No newline at end of file
+# mobile-menu
+
+![Microverse](https://img.shields.io/badge/-Microverse-6F23FF?style=for-the-badge)
+
+## Description
+
+This is a simple `Portfolio` project to demonstrate coding best practices including but not limited to understanding Gitflow, creating different base branches, understanding version control, and writing descriptive PRs and commit messages..
+
+
+## Library Directory 📙
+
+| Contents                    |
+| --------------------------- |
+| [Screenshot](#screenshot)   |
+| [Built With](#built-with-🛠) |
+| [Getting Started](#getting-started-🛠) |
+| [Contributing](#contributing🛠) |
+| [Authors](#authors)         |
+| [License](#license)         |
+
+
+ ## Screenshot
+
+![img](./mockup)
+
+## Built With 🛠
+
+```
+- HTML
+- CSS
+- JAVASCRIPT
+```
+
+## Getting Started
+
+- Clone the repo `https://github.com/keneogu/mobile-menu.git`
+- `cd mobile-menu`
+- `git checkout -b feature`
+- `git pull origin feature`
+
+## Contributing
+
+- Fork the project
+- Create your feature branch `git checkout -b awesome-feature`
+- Commit your changes `git commit -m 'Awesome feature'`
+- Push it `git push -u origin awesome-feature`
+- Open a pull request using this branch
+
+## Authors
+
+### 👨‍💻 Uduak John
+
+[![GitHub](https://img.shields.io/badge/-GitHub-000?style=for-the-badge&logo=GitHub&logoColor=white)](https://github.com/keneogu) <br>
+[![LINKEDIN](https://img.shields.io/badge/-LINKEDIN-0077B5?style=for-the-badge&logo=Linkedin&logoColor=white)](https://www.linkedin.com/in/kene-ogu/) <br>
+[![TWITTER](https://img.shields.io/badge/-TWITTER-1DA1F2?style=for-the-badge&logo=Twitter&logoColor=white)](https://twitter.com/keneogu)
+
+### 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/keneogu/mobile-menu).
```

Comment:
- [ ] [OPTIONAL] It should be better to have a valid link to your own GitHub's issues page! Please check and fix it.